tests/unittest.py

```diff
@@ -102,6 +103,10 @@ class TestCase(unittest.TestCase):
     use_memory_storage: bool = USE_MEMORY_STORAGE
     seed_config: Optional[int] = None
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._settings = get_settings()
```

Comment:
Why not do it in the `setUp()` method?

Reply:
Could be, but in both cases the same settings instance will be used for all tests. Doing it in `setUp()` would just override the same instance with itself. After this whole migration is complete and we begin to use injected settings, tests will be changed so each test can use its own custom settings instance, so we'll remove this.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/768/commits/d6e19ea3591bc301f5ee5be5e251326c0d1d08de

Reply:
Thanks!